pyproject.toml

```diff
@@ -16,6 +16,7 @@ numpy = "^1.23.0"
 setuptools = "^67.5.1"
 mypy = "^1.1.1"
 pytest-cov = "^4.1.0"
+tomlkit = "^0.13.2"
```

Comment:
Why do you want to use tomlkit instead of build in toml?

Reply:
Because `tomllib` is only for reading and we wanna write toml files. Additionally `tomlkit` keeps formatting when reading -> manipulating -> writing toml files, which is exactly what i want here. The `meta.toml` that the user specifies is merely a template where the skeleton id needs to be inserted.